### DISCUSSION TITLE
[allure-jest][ts] how to use allure.step, example from docs doesn't work

### DISCUSSION THREAD

**Discussion by vivregenek** [allure-framework/allure2#2673] [Questions & Support] [answered]
Hello,

Im using allure-jest 3.0.0-beta.9. Would like to specify steps, like its written in docs, but allure obj can't be found, I can't find it exported either by allure-jest or allure-js-common. 
``` 
it("Test Authentication", async () => {
  await allure.step("Step 1", async (s1) => {});
```
Also noticed in allure-js-commons/types/index.d.ts export from ./facade.js, if I change to export from ./facade.d.ts i can use exported by this 'step' function

**Comment by baev** [comment, marked as answer]
You can import `step` function from `allure-js-commons`:

```ts
import { step } from "allure-js-commons";

it("Test Authentication", async () => {
  await step("step 1", async () => {
    await step("step 1.1", async () => {
    });
    await step("step 1.2", async () => {
    });
  });
  ...
});
```

Or you can import all functions as `allure` alias:

```ts
import * as allure from "allure-js-commons";

it("Test Authentication", async () => {
  await allure.step("step 1", async () => {
    await allure.step("step 1.1", async () => {
    });
    await allure.step("step 1.2", async () => {
    });
  });
  ...
});
```

The documentation will be updated together with the release.
